PF1/Assignment2/RemoveDuplicate.java

```diff
@@ -0,0 +1,34 @@
+package assignment2;
+// Program to remove duplicate element from array by assigning max value to duplicate elements
+public class RemoveDuplicate {
+	 int[] removeDuplicate(int input[])
+	 {
+		 int max=Integer.MAX_VALUE ;
+		
+	   // assigning max value of integer to duplicate element in array
+	   for(int i=0;i<input.length;i++){
+		 for(int j=i+1;j<input.length;j++)
+		 {
+			 if(input[i]==input[j]){
+				input[j]=max; 
+			 }
+		 }
+	   } 
+	 //dup_array to store array values other than max value	 
+	  int dup_array[]=new int[input.length];
```

Comment:
variable declaration should be on top of the start of scope of either class in case of instance variables or method in case of local variables